src/CSV_Parser.php

```diff
@@ -42,6 +46,7 @@ class CSV_Parser {
 	 */
 	public function __construct() {
 		$this->setData(null);
+		$this->setEncoding(null);
```

Comment:
Should this be set to a default encoding instead of NULL i.e. UTF-8 or ISO-8859-1?